frontend/src/metabase/query_builder/components/expressions/TokenizedInput.jsx

```diff
@@ -124,8 +116,8 @@ class TokenizedInput extends Component {
           return;
         } else if (
           isSelected &&
-          ((atEnd && e.keyCode === KEYCODE_LEFT) ||
-            (atStart && e.keyCode === KEYCODE_RIGHT))
+          ((atEnd && e.key === "ArrowLeft") ||
+            (atStart && e.key === "ArrowRight"))
```

Comment:
There are a few changes from `keyCode` to `key`. Was there any particular reason to do so?

Reply:
Just a very very slight simplification.

Using `.key` we no longer need to declare 4 constants at the top, while keeping the conditionals (imo) just as readable if not more.

Reply:
Related: https://github.com/metabase/metabase/issues/14160